billy-spain/src/test/java/com/premiumminds/billy/spain/test/services/export/TestESCreditNotePDFTransformer.java

```diff
@@ -169,11 +169,10 @@ private ESCreditNoteEntity generateESCreditNote(PaymentMechanism paymentMechanis
 
         this.createSeries(reference, "AC");
 
-        ESCreditNoteEntity creditNote = null;
-        creditNote = (ESCreditNoteEntity) services.issueDocument(
-                new ESCreditNoteTestUtil(ESAbstractTest.injector).getCreditNoteBuilder(reference), params);
+        ESCreditNoteEntity creditNote = (ESCreditNoteEntity) services.issueDocument(
+            new ESCreditNoteTestUtil(ESAbstractTest.injector).getCreditNoteBuilder(reference), params);
```

Comment:
In order to make the code more readable, i'd suggest seperating each issueDocument parameter per line